docs/notebooks/libs/imports.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function
+import os
+import re
+import glob
+
+import numpy as np
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+import matplotlib.cm as cm
+import matplotlib.animation as animation
+from matplotlib import rc
+from matplotlib import colors
+
+from matplotlib.patches import Polygon as ppolygon
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+from matplotlib.ticker import MultipleLocator, FormatStrFormatter
+from matplotlib.animation import FuncAnimation
+from mpl_toolkits.mplot3d import Axes3D
+
+from xml.dom.minidom import parse
+
+import mpl_toolkits.axes_grid1
+import matplotlib.widgets
+
+from IPython.display import HTML
+
+from ipywidgets import interact, interactive, fixed, interact_manual
+import ipywidgets as widgets
+from ipywidgets import IntSlider
+
+from utils import * 
+
+
```

Comment:
This collection is indeed beautiful, but the understanding is only limited. Which package is required for which function?

Reply:
Sorry, I don't understand what you mean. Could you please be more specific?

Reply:
It is not clear to me which package is required for which part of the notebook collection. I suggest to add appropriate references as comments.

Reply:
﻿﻿Okay, some imports were not needed. I hope this is clearer now.
6bca149

Reply:
👍 , thanks